src/main/scala/fly/play/jiraExceptionProcessor/ReportingSupervisorStrategy.scala

```diff
@@ -42,15 +35,25 @@ class ReportingStrategyWrapper(wrapped: SupervisorStrategy, comment: String) ext
 
   private def reportException(context: ActorContext, exception: Throwable) = {
     val configuration = Configuration(context.system.settings.config)
+
     val wsClient = {
       val environment = Environment.simple(mode = Mode.Prod)
-      val wsConfig = new WSConfigParser(configuration, environment).parse
-      val ningWsConfig = new NingWSClientConfigParser(wsConfig, configuration, environment).parse
-      NingWSClient(ningWsConfig)
+
+      val parser = new WSConfigParser(configuration, environment)
+      val config = new AhcWSClientConfig(wsClientConfig = parser.parse())
+      val builder = new AhcConfigBuilder(config)
+      val ahcConfig = builder.configure().build()
+
+      implicit val system = ActorSystem()
+      implicit val materializer = ActorMaterializer()
+      new AhcWSClient(ahcConfig)
     }
 
     import context.dispatcher
 
-    new JiraExceptionProcessor(wsClient, configuration).reportError(ErrorInformation(exception, comment))
+    new JiraExceptionProcessor(wsClient, configuration)
+      .reportError(ErrorInformation(exception, comment))
+
+    wsClient.close()
```

Comment:
Instead of calling `close` here manually I recommend using the 'loaner pattern':

```scala
def withWsClient[T](code: WSClient => T): T = {
  val client = ...
  try code(client)
  finally client.close()
}
```

And usage:

```scala
withWSClient { wsClient =>
  ...
}
```

This has the added benefit of closing the client even when reporting the error fails.